streams/src/main/java/org/infinispan/tutorial/simple/streams/InfinispanStreams.java

```diff
@@ -24,9 +20,8 @@ public static void main(String[] args) {
       int range = 10;
       IntStream.range(0, range).boxed().forEach(i -> cache.put(i + "-key", i + "-value"));
       // Map and reduce the keys
-      int result = cache.keySet().stream()
-         .map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
-              .collect(() -> Collectors.summingInt(i -> i.intValue()));
+      int result = cache.keySet().stream().map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
```

Comment:
I would keep the `map` on a new line. As I mentioned before, I prefer aligning the chained method calls on the `.` on the first line, but it's not required if you don't like it :)

E.g.
```java
int result = cache.keySet().stream()
                  .map(e -> Integer.valueOf(e.substring(0, e.indexOf("-"))))
                  .collect(() -> Collectors.summingInt(i -> i.intValue()));
```